### DISCUSSION TITLE
PoitnLayer 如何根据数据配置不同的rotation

### DISCUSSION THREAD

**Discussion by dinglei19** [antvis/L7#2169] [Q&A]
const imageLayer = new PointLayer({ layerType: 'fillImage', })
      .source(carPoints, {
        parser: {
          type: 'json',
          x: 'longitude',
          y: 'latitude',
        },
      })
      .shape('car')
      .style({
        rotation: (data) => data.rotation, // 类似这种回调
        billboard: false,
      })
      .size(20);

**Comment by lzxue** [comment]
https://l7.antv.antgroup.com/examples/point/text/#point_text

可以的，看这个demo ,
